router.go

```diff
@@ -126,9 +129,14 @@ func createVersion(c echo.Context) (err error) {
 	if err = checkAuthorized(c); err != nil {
 		return err
 	}
+	space := getSpace(c)
+	prefix := space.Prefix
+	if prefix == "" {
+		prefix = consts.DefaultSpacePrefix
+	}
```

Comment:
I think we have a helper for this, no?

Reply:
Not in this PR. We'll have to rebase this current branch or replace this behaviour once the PR is merged to master.